roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -47,3 +47,26 @@ openshift_logging_mux_ops_ca: /etc/fluent/keys/ca
 #mux_config_contents:
 #mux_throttle_contents:
 #mux_secureforward_contents:
+
+# Default value of mux BUFFER_TYPE
+openshift_logging_mux_buffer_type: "file"
+
+# One of ['emptydir', 'pvc', 'hostmount']
+openshift_logging_mux_file_buffer_storage_type: "emptydir"
+
+# mount path: shared by hostmount and pvc
+openshift_logging_mux_file_buffer_mount_path: /mux/filebufferstorage
```

Comment:
For POSIX convention sake should we make this something like `/var/log/[fluentd|fluentd/mux|mux]/filebufferstorage`